### PR TITLE
Basic model (mongoose schema) for the meeting has been written.

### PR DESCRIPTION
**Pull request by LKhyeon**
### What this PR does (required):
- Creates a new model for Meeting.

### Screenshots / Videos (required):
- N/A

### Any information needed to test this feature (required):
- N/A

### CODE REVIEW

File: server/models/Meeting.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const meetingSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true
+  },
```
Comment (moffatethan) [inline server/models/Meeting.js]: Here we are saying any string can be a user id where we want to reference another object in the users collection. Mongoose provides a schema type for ObjectId. `mongoose.Types.ObjectId` and make sure to `ref: 'user'`. Also, JavaScript convention is camelCase and not snake_case.

**Review by moffatethan — COMMENTED** [review]
Left you a comment to address, otherwise looks good!

File: server/models/Meeting.js (hunk 1 above)
Comment (LKhyeon) [reply server/models/Meeting.js]: I've changed tthe type of `user_id` to `mongoose.Types.ObjectId` and added `ref: "user"`.

File: server/models/Meeting.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require("mongoose");
+
+const meetingSchema = new mongoose.Schema({
+  user_id: {
```
Comment (moffatethan) [inline server/models/Meeting.js]: Should be `userId`

**Review by moffatethan — APPROVED** [review]
LGTM!

File: server/models/Meeting.js (hunk 2 above)
Comment (LKhyeon) [reply server/models/Meeting.js]: Yes, this is my bad - I forgot about this specific feedback. I have made appropriate changes to it.

**Pull request merged**